.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,23 @@
+name: unit-tests
+
+on:
+  pull_request:
+    branches:
+      - main
```

Comment:
Can you change this so that the test run more often as seen in GH collab unit tests [yml file](https://github.com/ministryofjustice/github-collaborators/blob/89c3009fb7975c3408301b646dee4a7a84a47869/.github/workflows/rspec.yml#L10). At the moment it will only run the test when a PR to main is created, so it may catch errors at the last minute before ready to merge rather than earlier